lib/Wikia/src/DependencyInjection/InjectorBuilder.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * InjectorBuilder
+ *
+ * <insert description here>
```

Comment:
Ooops, that's IntelliJ auto generated stuff when adding a new file.

On Wed, Jun 24, 2015 at 1:30 PM, Damon Snyder <notifications@github.com>
wrote:

> In lib/Wikia/src/DependencyInjection/InjectorBuilder.php
> <https://github.com/Wikia/app/pull/7631#discussion_r33192236>:
>
> > @@ -0,0 +1,56 @@
> > +<?php
> > +/**
> > + * InjectorBuilder
> > + *
> > + * <insert description here>
>
> Minor: does it make sense to add some header documentation to these?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/7631/files#r33192236>.
>